pkg/tests/tasks/migration/migration_test.go

```diff
@@ -90,45 +114,158 @@ spec:
 		oc.DefaultOC.WaitFor(t, smcp.Namespace, "ServiceMeshMemberRoll", "default", `jsonpath='{.status.configuredMembers[?(@=="bookinfo")]}'`)
 
 		t.LogStep("Create 3.0 controlplane and IstioCNI")
-		istioTempl := `apiVersion: sailoperator.io/v1alpha1
-kind: Istio
+		setupIstio(t, istio)
```

Comment:
Should we install istio only after the workloads are up and running to follow the step migration step order?